.babelrc

```diff
@@ -1,9 +1,13 @@
 {
   "plugins": [
-    "transform-object-rest-spread",
-    "transform-object-assign",
-    "transform-class-properties",
+    ["@babel/plugin-transform-flow-strip-types", {"requireDirective": true}],
```

Comment:
Babel seems to barf up on files like PrefixIntervalTree.js, due to flow annotations.
Using this plugin in to strip them off.

Reply:
A

Reply:
> Babel seems to barf up on files like PrefixIntervalTree.js, due to flow annotations.
> Using this plugin in to strip them off.

Reply:
A